integration/presto/src/main/java/org/apache/carbondata/presto/impl/CarbonTableConfig.java

```diff
@@ -40,6 +40,12 @@
   private String endPoint;
   private String pushRowFilter;
 
+  /**
+   * Property to send load model from coordinator to worker in presto. This is internal constant
+   * and not exposed to user.
```

Comment:
describe what are the option values

Reply:
As said in the above comment, its same we use this as property name to send the load model from coordinator to worker. So its value will be the load model prepared for each load.